libmamba/tests/src/core/test_mirror.cpp

```diff
@@ -10,7 +10,7 @@
 
 namespace mamba
 {
-    TEST_SUITE("mirror")
+    /*TEST_SUITE("mirror")
```

Comment:
Is there a reason the tests are commented here?

Reply:
Yes, I have started to partially move the mirror implementation to the download folder / namespace. Since I want to make incremental steps, I have not port everything yet, but I had to remove `core/mirror.cpp` from the CMakeLists.txt to avoid symbol duplication, and I had to comment this test until the missing functions are ported to `download`